### DISCUSSION TITLE
Need help to configure GIT with SSH

### DISCUSSION THREAD

**Discussion by fslef** [requarks/wiki#3097] [Help / Questions]
Hello all,

I've installed my Wiki.JS instance on Azure using the tuto available:
- https://docs.requarks.io/install/azurewebapp

I try now to configure the GIT Storage. For security reasons, I would like to use SSH.
So I followed up the:
- https://docs.requarks.io/storage/git Tuto.

Here is a resume of what I did:
- SSH key is generated from the Git Bash
- Public Key has been uploaded to the repo config
- Private key is specified in the Wiki.JS Config using the B option: B - SSH Private Key Content"

For the configuration, I always proceeding this way:
1. GIT Connector configuration (but not enabled)
2. Enable the connector

When I enable the Git storage, I run into the following error:

`Warning: Identity file not accessible: No such file or directory. git@github.com: Permission denied (publickey). fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists. error: Could not fetch origin`

Could you please help me to troubleshoot ?

**Comment by fslef** [comment]
Nobody has an idea ?

**Reply by justinacolmena** [reply]
Sorry, I'm not an expert, but I have used another app on GitHub, https://github.com/android-password-store/Android-Password-Store, which has a feature to save GPG-encrypted passwords to a GIT repository by SSH -- I have tried it with varying levels of success by setting up a personal GIT repository on a VPS, but that's written in Java, not JavaScript, as far as I can tell.
